projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -904,3 +904,18 @@ function get_paywall_simple() {
 <!-- /wp:columns -->
 ';
 }
+
+/**
+ * Determine if the newsletter categories are enabled for the site.
+ *
+ * This function checks whether the blog has the 'newsletter_categories' sticker enabled.
+ * If the sticker is enabled, the function will return true.
+ * If not, it applies the 'jetpack_newsletter_categories' filter and returns the result (default is false).
+ */
+function newsletter_categories_enabled_for_site() {
+	if ( function_exists( 'has_blog_sticker' ) && has_blog_sticker( 'newsletter_categories' ) ) {
+		return true;
+	}
+
+	return apply_filters( 'jetpack_newsletter_categories', false );
```

Comment:
Maybe that hook should be a bit more verbose, so folks know what it exactly does (enable the feature)?

Reply:
> Maybe that hook should be a bit more verbose, so folks know what it exactly does (enable the feature)?

Perhaps `jetpack_newsletter_categories_enabled`?

Looking at the [Codex](https://developer.jetpack.com/) all forms of `disabled/enabled/disable/etc.` suffixing/prefixing are used so there's no set convention.

Reply:
> Perhaps jetpack_newsletter_categories_enabled?

I like it 👍